packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -3250,10 +3250,10 @@
   "RetentionPolicy_MaxAge_Groups": "Maximum message age in private groups",
   "RetentionPolicy_Precision": "Timer Precision",
   "RetentionPolicy_Precision_Description": "How often the prune timer should run. Setting this to a more precise value makes channels with fast retention timers work better, but might cost extra processing power on large communities.",
-  "RetentionPolicy_RoomWarning": "Messages older than __time__ are automatically pruned here",
-  "RetentionPolicy_RoomWarning_FilesOnly": "Files older than __time__ are automatically pruned here (messages stay intact)",
-  "RetentionPolicy_RoomWarning_Unpinned": "Unpinned messages older than __time__ are automatically pruned here",
-  "RetentionPolicy_RoomWarning_UnpinnedFilesOnly": "Unpinned files older than __time__ are automatically pruned here (messages stay intact)",
+  "RetentionPolicy_RoomWarning": "Messages older than __time__ days are automatically pruned here",
```

Comment:
Maybe, to reflect that __time__ can be `1` we can write `day(s)` here 🤔 not sure